src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -276,6 +276,24 @@ and a guide for using macros can be found at
 
 All reconstruction-specific parameters that are not specified, but one wants to include, should go into the `ReconMethodParameterValues` field.
 
+#### Task
+
+<!-- This block generates a metadata table.
+The definitions of these fields can be found in
+  src/schema/objects/metadata.yaml
+and a guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_metadata_table(
+   {
+      "TaskName": ("RECOMMENDED", "If used to denote resting scans, a RECOMMENDED convention is to use labels beginning with `rest`.")
```

Comment:
Usually this is REQUIRED for almost all datatypes (except for behavioral), for backward compatibility I suspect it is better to keep it RECOMMENDED.

Reply:
Added an extra comment that tries to make it clear that eventhough task is not required for PET for resting scans, if one decided to use it, it would be better to stick to the convention other datatypes use.

Reply:
I agree that it is best to keep recommended, and great that you added the extra comment.